saleor/shipping/interface.py

```diff
@@ -30,6 +30,8 @@ class ShippingMethodData:
     minimum_delivery_days: Optional[int] = None
     metadata: Dict[str, str] = field(default_factory=dict)
     private_metadata: Dict[str, str] = field(default_factory=dict)
+    active: bool = True
+    message: Optional[str] = ""
```

Comment:
Does it make sense to allow None here? Won't an empty string be sufficient?

Reply:
I believe you're right, allowing None when assigning an empty string either way seems redundant